src/vs/workbench/parts/scm/browser/scmPreview.ts

```diff
@@ -14,18 +14,11 @@ import { IMessageService } from 'vs/platform/message/common/message';
 import pkg from 'vs/platform/node/package';
 // tslint:enable
 
-// Enable this by default for insiders
-function getDefaultValue(): boolean {
-	const value = window.localStorage.getItem('enablePreviewSCM');
-	return /insider$/.test(pkg.version) ? value !== 'false' : value === 'true';
-}
-
 export default class SCMPreview {
 
-	private static readonly _enabled = getDefaultValue();
-
 	static get enabled(): boolean {
-		return this._enabled;
+		// We want SCM to always be enabled on sourcegraph.com.
+		return true;
```

Comment:
Should we really always modify VS Code internals when we want to enable _default behavior_ that can also be set through a setting? This will make it hard to merge upstream VS Code the more we do it